src/components/templates/WalletTransactionsAndAssets/WalletTransactionsAndAssets.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { View, ScrollView, Dimensions } from 'react-native';
+import { Button, Row, Spacer, Text } from '@components/base';
+import { verticalScale } from '@utils/scaling';
+import { COLORS } from '@constants/colors';
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming
+} from 'react-native-reanimated';
+import { SingleAsset } from '@components/templates/WalletTransactionsAndAssets/WalletAssets/SingleAsset';
+import { styles } from '@components/templates/WalletTransactionsAndAssets/styles';
+import { WalletTransactions } from '@components/templates/WalletTransactionsAndAssets/WalletTransactions';
+import {
+  useCryptoAccountFromHash,
+  useSelectedWalletHash,
+  useTransactionsOfAccount
+} from '@hooks';
+import { useTranslation } from 'react-i18next';
+import { SingleTransaction } from '@components/templates/WalletTransactionsAndAssets/WalletTransactions/SingleTransaction';
+import { WalletAssets } from '@components/templates/WalletTransactionsAndAssets/WalletAssets';
+
+interface WalletTransactionsAndAssetsProps {
+  address: string;
+}
+
+export const WalletTransactionsAndAssets = (
+  props: WalletTransactionsAndAssetsProps
+) => {
+  const { address } = props;
+  const scrollView = useRef<ScrollView>(null);
+  const selectedWalletHash = useSelectedWalletHash();
+  const { data: account, loading: accountLoading } =
+    useCryptoAccountFromHash(selectedWalletHash);
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const { t } = useTranslation();
+
+  const initialMount = useRef(true);
+  const TRANSACTION_LIMIT = 20;
+  const { data: transactions, loading: transactionsLoading } =
+    useTransactionsOfAccount(
```

Comment:
I think we should not use our own endpoint to get transactions of an address. We won't be able to support other tokens in the future once we add them. We better use a third party API, like etherscan etc. 
Let's see how TrusteeWallet handles it